### DISCUSSION TITLE
Congratulations! WingetUI is growing quite fast (and it can still grow a lot more!)

### DISCUSSION THREAD

**Discussion by ppvnf** [Devolutions/UniGetUI#342] [General]
Version 1.4.2 got almost 18k downloads (in 21 days) and the repo has almost 700 stars!

For comparison, WingetUI 1.3.1 had 11k downloads in 26 days and the repo had less than 600 hundred stars last month!

I've tested the screenshots/translation update and I am really excited about WingetUI future. I wish I could help more by doing some web design, but I don't know how to do it. 

I don't think there's anything bad about WingetUI design. it's in fact better than I could have ever imagined. However, there's still some room for improvement. The following points are just my personal opinion:

The screenshots are nice, but I believe some explanation together with the images would be better. WingetUI icon isn't bad, but it's a little bit generic. Also, I am not sure if a website would help people find/trust WingetUI or if it would be useless

**Comment by Spacellary** [comment]
Hurray! 🎉
I also gave the new features a try and it's definitely heading in a great direction.
Too bad I don't have any experience coding but I'm at least cheering on the sidelines~

Spicing it would be nice, we could vote on ideas. The WingetUI icon at least does match the style of the other Windows 11 applications.
Not sure if a website would be necessary or do much in terms of visibility, at least not for now. I do know that some people find GitHub weird at first and if they aren't used to the website, it's definitely not "users first", but I think the current Read Me does a good enough job.

**Comment by marticliment** [comment]
About the website, i really think with the GitHub repo it's just enough (while a website wouldn't be useless at all), and about the icon, i understand your arguments, and it might change in a future. 

And if you are liking WingetUI, stay tuned because WingetUI version 1.5.0 is just around the corner, and will be published (hopefully) in the following days!

And also, thatk you all, without you this wouldn't have been possible

**Reply by ppvnf** [reply]
> And if you are liking WingetUI, stay tuned because WingetUI version 1.5.0 is just around the corner, and will be published (hopefully) in the following days!

That's good to know! The only must fix bug left that I am aware is this one: https://github.com/martinet101/WingetUI/issues/307

**Reply by marticliment** [reply]
1.5.0 is already out!
